backend/src/api/index.js

```diff
@@ -4,12 +4,14 @@ const futureInterview = require('./futureInterview')
 const matchCandidates = require('./matchCandidates')
 const matches = require('./matches')
 const structure = require('./structure')
+const events = require('./events')
 
 module.exports = {
   candidates,
   interview,
   futureInterview,
   matchCandidates,
   matches,
-  structure
+  structure,
+  events
```

Comment:
Also alphabetical order

Reply:
no need for alphabetical order stuff right now. The current code doesn't adhere to it, and it doesn't make sense to address those kind of code quality changes in a feature PR. Doing those kind of changes can happen at a different time, but right now it would lead to a rabbit hole across the whole application.